### PR TITLE
Style: sync linters & typecheckers

### PR DESCRIPTION
**Pull request by ctizen**

### CODE REVIEW

File: packages/public-api/tsconfig.build.json [hunk 1]
```diff
@@ -2,7 +2,10 @@
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "outDir": "./build",
-    "module": "commonjs"
+    "module": "commonjs",
+    "paths": {
+      "@gradejs-public/shared": ["../shared/src"]
```
Comment (spalt08) [inline packages/public-api/tsconfig.build.json:7]: It'll probably break the deploy logic. We should use path aliases only in the development mode (specified in `tsconfig.json` in the root). Production configs should not use path aliases for monorepo packages

File: packages/web/src/index.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import ReactGTM from 'react-gtm-module';
 import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import App from 'components/App';
+// eslint-disable-next-line import/extensions
```
Comment (spalt08) [inline packages/web/src/index.tsx:6]: This rule may be quite inconvenient for the front-end package

File: packages/worker/tsconfig.build.json [hunk 3]
```diff
@@ -2,7 +2,10 @@
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "outDir": "./build",
-    "baseUrl": "./src"
+    "baseUrl": "./src",
+    "paths": {
+      "@gradejs-public/shared": ["../../shared/src"]
```
Comment (spalt08) [inline packages/worker/tsconfig.build.json:7]: The same here

**Review by spalt08 — CHANGES_REQUESTED** [review]

File: packages/web/src/index.tsx (hunk 2 above)
Comment (ctizen) [reply packages/web/src/index.tsx:6]: Good point, fixed in much more proper way.

File: packages/worker/tsconfig.build.json (hunk 3 above)
Comment (ctizen) [reply packages/worker/tsconfig.build.json:7]: Good point, fixed in much more proper way.

**Review by spalt08 — APPROVED** [review]

**Pull request merged**